### DISCUSSION TITLE
How to identify builds

### DISCUSSION THREAD

**Discussion by GHNewbiee** [fortran-lang/fpm#794] [General]
In the *build* directory I get a new sub-directory each time I build a project with different options. How can I find out after a while which one is build with what options? *Tia*

**Comment by zoziha** [comment]
You can use the following command to see the folder name with the specific hash value:
1. `fpm build --flag "xxx" --list`
2. `fpm run --flag "xxx" --runner du`
3. `fpm build -V`

Hope this helps.

**Comment by GHNewbiee** [comment]
@zoziha *TfyR&T*

I see that every build includes an empty `<project>.log` file in the `.../build/xxx.../app` dir. It would be helpful if the first line was something like:

`<build time stamp> - <full cmd> or <build options(only)>`

For someone working with IDE, opening the file is very quick and easy.

It would be even more helpful if there was a single `<project>.build.log` file in `.../build` dir which includes all builds:

```
<time stamp> - gfortran_1x.. -  <full cmd> or <build options(only)>
<time stamp> - gfortran_2x.. -  <full cmd> or <build options(only)>
...
```
